internal/pkg/test/tool/require/require.go

```diff
@@ -258,7 +259,7 @@ func Filesystem(t *testing.T, fs string) {
 // in one the path defined in the PATH environment variable,
 // if not found the current test is skipped with a message.
 func Command(t *testing.T, command string) {
-	_, err := exec.LookPath(command)
+	_, err := bin.FindBin(command)
```

Comment:
Might need to check that this doesn't skip more tests.

We are quite likely to be using `require.Command` for something in a test that *isn't* expected to be used by Singularity.

Reply:
Thanks, good point - changing it so that it tries bin.FindBin() and if that fails, tries exec.LookPath(). Only if both fail, will t.Skipf() actually be called.